app/main.py

```diff
@@ -1,13 +1,41 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "column": error["column_number"], "line": error["line_number"],
+        "message": error["text"], "name": error["code"], "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [
+            {
+                "line": error["line_number"],
+                "column": error["column_number"],
+                "message": error["text"],
+                "name": error["code"],
+                "source": "flake8"
+            } for error in errors
+        ],
+        "path": file_path, "status": "failed"
```

Comment:
What about "passed" status? It's possible that there are no errors for same file, thus status should be "passed". It's mentioned in the requirements ;)